x-pack/test/stack_functional_integration/apps/ccs/ccs_console.js

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+
+export default function ({ getService, getPageObjects }) {
+  const retry = getService('retry');
+  const log = getService('log');
+  const PageObjects = getPageObjects(['common', 'console']);
+
+  describe('Integration Tests - Console App CCS', function describeIndexTests() {
+    this.tags('includeFirefox');
```

Comment:
We don't run Firefox browser config (anymore) in integration-test.  But this won't hurt anything